### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ioeides**
- add option 224 which is definitely mandatory for gen 2 RFPs.
- some minor changes on phrasing

### CODE REVIEW

**Review by blackdotraven — APPROVED** [review]
looks good to me

no test happened on my side to verify anything. Just looking at the diff.

**Comment by ioeides** [comment]
looks like one have to set tftp boot file name for gen 2 rfps

**Comment by blackdotraven** [comment]
Does this tftp Stuff work without setting an TFTP-Server as some kind of dhcp-option? Looks like dnsmasq sets it self on default?

File: index.md [hunk 1]
```diff
@@ -312,9 +312,10 @@ dhcp-authoritative
 dhcp-range=10.10.10.100,10.10.10.199,255.255.255.0,12h
 
 # static leases for RFPs
-dhcp-host=00:30:42:cc:23:42,10.10.10.10,rfp-1
-dhcp-host=00:30:42:cc:23:43,10.10.10.11,rfp-2
-dhcp-host=00:30:42:cc:23:44,10.10.10.12,rfp-3
+# for every RFP, set a tag to choose on which tftp boot file to take (tags are definded below: rfp2g/rfp3g/rfp3g)
```
Comment (blackdotraven) [inline index.md:315]: rfp2g/rfp3g/**rfp3g**
rfp2g/rfp3g/**rfp4g**

**Review by blackdotraven — CHANGES_REQUESTED** [review]

**Review by blackdotraven — APPROVED** [review]

File: index.md [hunk 2]
```diff
@@ -302,37 +302,50 @@ host myfancyrfp {
...
 server=9.9.9.9
 server=149.112.112.112
-```
 
+# enable built-in tftp server (e.g. for gen 2 boot files, ...)
+enable-tftp
+tftp-root=/srv/tftp
+```
 
-The RFP will ignore DHCP offers that do not contain the `magic_str`. This means that (1) the magic string must be included in DHCP replies intended for RFPs, so (2) it may be possible to have two co-existing DHCP servers (one for normal devices and one for RFPs).
```
Comment (n-st) [inline index.md:335]: @vschlegel Was it intentional that you removed this part?
(I see that it's mentioned in a comment in the config section, but people might miss it if it's not in the "plaintext" part of the how-to)

Comment (Janik-Haag) [reply index.md:335]: I would assume someone using dhcpd won't look at the dnsmasq config and the other way around. But we could put it into it's own section and reference it in the config to save on duplicate effort maintaining.

**Pull request merged**